### PR TITLE
DATAUP-246: misc css fixes

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of changes

- Removing unreferenced css files
- Moving inline styles to stylesheets
- Renaming `landing-pages.css` to `landingPages.css`
- Tidying up font declarations
- Using minified css for third-party libs
- Removing aberrant font declaration that was making sonar cloud complain

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-246

# Testing Instructions
* Details for how to test the PR: 
- [ ] Tests pass in travis and locally 
- [ ] Changes available by spinning up a local narrative and clicking around to see that nothing has changed

# Dev Checklist:

- [ ] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: kbase-extension/kbase_templates/narrative_header.html [hunk 1]
```diff
@@ -73,7 +73,6 @@
                         <li class="divider"></li>
 
                         <li title="About Jupyter Notebook"><a id="notebook_about" href="#">About Jupyter</a></li>
-                        <li title="About IPywidgets" style="display:none"><div style="display:none"><a id="help_menu" href="#"></a></div></li>
```
Comment (ialarmedalien) [inline kbase-extension/kbase_templates/narrative_header.html:76]: this line is not displayed ==> delete it

File: kbase-extension/kbase_templates/page.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <link rel="stylesheet" href="{{ static_url("kbase/css/kbaseStylesheet.css") }}" type="text/css" />
     <link rel="stylesheet" href="{{ static_url("kbase/css/kbaseNarrative.css") }}" type="text/css" />
     <link rel="stylesheet" href="{{ static_url("kbase/css/kbaseIcons.css") }}"  type="text/css"/>
-    <link rel="stylesheet" href="{{ static_url("kbase/css/landing-pages.css") }}" type="text/css" />
+    <link rel="stylesheet" href="{{ static_url("kbase/css/landingPages.css") }}" type="text/css" />
```
Comment (ialarmedalien) [inline kbase-extension/kbase_templates/page.html:20]: renamed this file to match the others in the directory

File: kbase-extension/static/kbase/css/appCell.css [hunk 3]
```diff
@@ -4,11 +4,11 @@
 */
 
 .kb-app-cell {
-  font-family: 'Oxygen', sans-serif;
+  font-family: Oxygen, Arial, sans-serif;
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/appCell.css:7]: standardised the font declarations and added in fallback fonts where they were absent

File: kbase-extension/static/kbase/css/kbaseNarrative.css [hunk 4]
```diff
@@ -258,6 +263,12 @@
 
 .kb-data-list-drag-target {
   transition: all 0.25s ease;
+  display: block;
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/kbaseNarrative.css:266]: combined stanzas

File: kbase-extension/static/kbase/css/kbaseNarrative.css [hunk 5]
```diff
@@ -271,17 +282,7 @@
...
-  display: block;
-  border: 2px orange dashed;
-  height: 40px;
-  text-align: center;
-  padding: 6px;
-  border-radius: 4px;
-}
-
 #kb-notify-area {
-  display: inline-block;
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/kbaseNarrative.css:284]: `display` property is overridden by `float: left`

File: kbase-extension/static/kbase/css/kbaseNarrative.css [hunk 6]
```diff
@@ -3478,18 +3483,23 @@ button.kb-data-obj {
...
+.kb-loading-blocker__text {
   width: 40%;
   margin-left: 30%;
   font-size: 16px;
   text-align: left;
-  line-height: 1.5em;
+  line-height: 1.5;
+}
+
+.kb-loading-blocker__image_container {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/kbaseNarrative.css:3500]: loading template - moved inline styles to the stylesheet

File: kbase-extension/kbase_templates/notebook.html [hunk 7]
```diff
@@ -16,15 +16,13 @@
 <link rel="stylesheet" href="{{ static_url("components/bootstrap-tour/build/css/bootstrap-tour.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("components/codemirror/lib/codemirror.css") }}">
 {{super()}}
-<link rel="stylesheet" href="{{ static_url("kbase/css/contigBrowserStyles.css") }}" type="text/css" />
```
Comment (ialarmedalien) [inline kbase-extension/kbase_templates/notebook.html:19]: moved this to be with the other kbase stylesheets in `page.html`

File: kbase-extension/kbase_templates/notebook.html [hunk 8]
```diff
@@ -16,15 +16,13 @@
...
 <link rel="stylesheet" href="{{ static_url("ext_components/datatables/media/css/dataTables.bootstrap.min.css") }}" type="text/css" />
-<link rel="stylesheet" href="{{ static_url("ext_components/select2/dist/css/select2.css") }}" type="text/css" />
+<link rel="stylesheet" href="{{ static_url("ext_components/select2/dist/css/select2.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("ext_components/select2-bootstrap-theme/dist/select2-bootstrap.min.css") }}" type="text/css" />
-<link rel="stylesheet" href="{{ static_url("ext_components/dropzone/dist/dropzone.css") }}" type="text/css" />
+<link rel="stylesheet" href="{{ static_url("ext_components/dropzone/dist/min/dropzone.min.css") }}" type="text/css" />
```
Comment (ialarmedalien) [inline kbase-extension/kbase_templates/notebook.html:22]: using minified versions of stylesheets

File: kbase-extension/static/kbase/css/landingPages.css [hunk 9]
```diff
@@ -351,11 +351,6 @@ table {
   padding-left: 20px;
   padding-right: 20px;
   border: 1px solid #428bca;
-
-  /* FIXME: why does the font change here?! */
-
-  font-family: OxygenBold;
-  font-size: 14px;
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/landingPages.css:358]: I can't even find anywhere that this style is used, so deleting this aberrant font declaration for `.active .btn-narr`.

**Comment by ialarmedalien** [comment]
SonarCloud bug is replacing an `<i>` with an `<em>` in one of the menus -- this was not code that I had touched so I will leave as-is.

File: kbase-extension/kbase_templates/notebook.html (hunk 7 above)
Comment (briehl) [reply kbase-extension/kbase_templates/notebook.html:19]: Good idea for now. I think that should be imported by the contig browser widget as requested (at least I think that's the only place it gets used). But that's a separate piece of work.

**Review by briehl — CHANGES_REQUESTED** [review]
Other than deleting that widget help element, which throws an error, this looks good.

**Review by briehl — APPROVED** [review]

**Review by eamahanna — APPROVED** [review]

**Pull request merged**